File: CT_BAEKJOON_11021.c
//https://www.acmicpc.net/problem/11021
//https://www.acmicpc.net/source/40092946

#include <stdio.h>
#pragma warning(disable:4996)
int main()
{
	int A , B ,C,  total ,h, m;
	scanf("%d",&B);

	A = 0;
	C = 0;

	for (int i = 1; i < B + 1 ; i++) {
		scanf("%d %d", &A, &C);
		printf("Case #%d: %d\n", i, A + C);

	}

}	

File: CT_BAEKJOON_11021.py
#https://www.acmicpc.net/problem/11021
#https://www.acmicpc.net/source/40143947
    
A = input()

A = int(A)

for i in range(1,A+1):
    B , C = input().split()
    B = int(B)
    C = int(C)
    print("Case #%d: %d" %(i , B + C))